src/run.rs

```diff
@@ -95,12 +111,40 @@ pub fn image_ids() -> io::Result<HashSet<String>> {
             output
                 .lines()
                 .map(|line| line.trim().to_owned())
-                .filter(|line| !line.is_empty())
+                .filter_map(|line| {
+                    if line.is_empty() {
+                        let tab_index = line.find('\t').unwrap();
+                        let (image_id, date_str) = line.split_at(tab_index);
+                        Some((image_id.to_owned(), parse_docker_date(date_str.trim())))
```

Comment:
Can we `trim` the `image_id` as well?

Reply:
I understand the other `trim` is necessary because of the tab character, whereas the `image_id` `trim` is probably not needed. But it would help me sleep at night if we trim everything that comes from the output of an external program.